src/databricks/labs/ucx/framework/tasks.py

```diff
@@ -1,11 +1,14 @@
 import logging
+import os
 from collections.abc import Callable
 from dataclasses import dataclass
 from functools import wraps
 from pathlib import Path
 
+from databricks.sdk.runtime.dbutils_stub import dbutils
```

Comment:
Don't import dbutils. We should not be using it in this project. Besides -what you imported is not a real dbutils instance

Reply:
Fixed